src/main/java/org/jitsi/stats/DataDogTags.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright @ 2018 Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.stats;
+
+/**
+ * This class saves tags which specify for which service an aspect
+ * is incremented when it uses
+ * {@link DataDogStatsClient#incrementCounter(String, String...)}
+ *
+ * @author Nik Vaessen
+ */
+public class DataDogTags
```

Comment:
i wonder if a centralized location of aspects/tags is best...since they are strings maybe we should just let users of this client define aspects/tags as they see fit?  at least for the aspects it would give some consistency (but can also result in using doesn't-quite-fit aspects purely for the purpose of reuse), but the tags seem to me like something that should be left to the calling code.